media-video/handbrake/handbrake-1.2.2.recipe

```diff
@@ -99,8 +110,10 @@ PATCH()
 
 BUILD()
 {
+	export CPPFLAGS="-I/system/develop/headers/libxml2"
```

Comment:
This is not a clean solution and would break on x86. Use pkgconfig.

Reply:
Fixed.